features/step_definitions/questions_steps.rb

```diff
@@ -36,7 +36,8 @@
   expect(page).not_to have_content(message)
 end
 
-When /I select Option "(.*)" for Question 1 for the Testing the Hide\/Show Functionality Survey/ do |option|
-  radio_button = page.all('input[type="radio"]').find { |radio| radio.value == option }
-  radio_button.click
-end
\ No newline at end of file
+# When /I select Option "(.*)" for Question 1 for the Testing the Hide\/Show Functionality Survey/ do |option|
```

Comment:
lets delete this, rather than comment out. Not needed after you added steps in features/step_definitions/responses_steps.rb